app/controllers/middleware_server_controller.rb

```diff
@@ -12,84 +12,109 @@ class MiddlewareServerController < ApplicationController
   after_action :set_session_data
 
   COMMON_OPERATIONS = {
-    :middleware_server_reload  => {:op   => :reload_middleware_server,
-                                   :skip => true,
-                                   :hawk => N_('reloading'),
-                                   :msg  => N_('Reload')
+    :middleware_server_reload  => {
+      :op           => :reload_middleware_server,
+      :log_timeline => 'MwServer.Reload.UserRequest',
+      :skip         => true,
+      :hawk         => N_('reloading'),
+      :msg          => N_('Reload')
     },
-    :middleware_server_suspend => {:op    => :suspend_middleware_server,
-                                   :skip  => true,
-                                   :hawk  => N_('suspending'),
-                                   :msg   => N_('Suspend'),
-                                   :param => :timeout
+    :middleware_server_suspend => {
+      :op           => :suspend_middleware_server,
+      :log_timeline => 'MwServer.Suspend.UserRequest',
+      :skip         => true,
+      :hawk         => N_('suspending'),
+      :msg          => N_('Suspend'),
+      :param        => :timeout
     },
-    :middleware_server_resume  => {:op   => :resume_middleware_server,
-                                   :skip => true,
-                                   :hawk => N_('resuming'),
-                                   :msg  => N_('Resume')
+    :middleware_server_resume  => {
+      :op           => :resume_middleware_server,
+      :log_timeline => 'MwServer.Resume.UserRequest',
+      :skip         => true,
+      :hawk         => N_('resuming'),
+      :msg          => N_('Resume')
     },
-    :middleware_dr_generate    => {:op   => :generate_diagnostic_report,
-                                   :skip => false,
-                                   :hawk => N_('generating JDR report'),
-                                   :msg  => N_('Generate JDR report')}
+    :middleware_dr_generate    => {
+      :op   => :generate_diagnostic_report,
+      :skip => false,
```

Comment:
@israel-hdez just curious if :log_timeline is missing in this block or there is no need to add one here

Reply:
No need to add one here. If it's missing, it just won't generate the timeline event. The ones that doesn't have `:log_timeline` aren't power operations ;)